scripts/l1_node.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
-set -exu
+# set -exu
+set -eu
 
-NETWORK="${NETWORK:-testnet}"
+NETWORK="${NETWORK:-edgenet}"                 #! Replace with your network name
 GENESIS_URL="https://raw.githubusercontent.com/allora-network/networks/main/${NETWORK}/genesis.json"
 SEEDS_URL="https://raw.githubusercontent.com/allora-network/networks/main/${NETWORK}/seeds.txt"
-BLOCKLESS_API_URL="${BLOCKLESS_API_URL:-https://heads.${NETWORK}.allora.network}"               #! Replace with your blockless API URL
 
-APP_HOME="${APP_HOME:-/data}"
+export APP_HOME="${APP_HOME:-./data}"
 INIT_FLAG="${APP_HOME}/.initialized"
 MONIKER="${MONIKER:-$(hostname)}"
 KEYRING_BACKEND=test                              #! Use test for simplicity, you should decide which backend to use !!!
 GENESIS_FILE="${APP_HOME}/config/genesis.json"
 DENOM="uallo"
 
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+"${SCRIPT_DIR}/restore_snapshot.sh"
```

Comment:
I am a little nervous about this line

l1_node is set as the entrypoint for `docker-compose.yaml` 

the local_testnet_l1.sh script uses docker compose, my only concern is I don't want the script that devs use locally to test their code for e.g. integration, stress tests, to be pushing out or pulling down s3 snapshots every time the local_testnet_l1.sh runs - especially if they don't have the keys/token to access s3

Are you sure this will restore_snapshot will not execute if a dev is running a local testnet?

Reply:
So we may just separete concerns maybe? initialy the docker-compose was meant to be a way for nops to run node easily right? if we now use that for integration test, then we probably have to provide another means by which nops run the node.

or maybe I can make it optional. i.e commented off and added if needed?

Reply:
Did the latter!